distribution/smarthome/conf/smarthome.cfg

```diff
@@ -26,4 +26,5 @@ com.eclipsesource.jaxrs.swagger.config:swagger.info.version=1.0
 # Configuration of thread pool sizes
 org.eclipse.smarthome.threadpool:thingHandler=3
 org.eclipse.smarthome.threadpool:discovery=3
-org.eclipse.smarthome.threadpool:safeCall=3
\ No newline at end of file
+org.eclipse.smarthome.threadpool:safeCall=3
+org.eclipse.smarthome.threadpool:astro=30
```

Comment:
This is just an example configuration and it is up to the solutions/user to provide a specific one. In general, the code should work smoothly with the default size (used when there is no configuration), which is 5.